src/python/WMCore/WMSpec/WMWorkloadTools.py

```diff
@@ -64,13 +64,8 @@ def strToBool(string):
 
 def _verifyDBSCall(dbsURL, uri):
     try:
-        # from WMCore.Services.DBS.DBS3Reader import DBS3Reader
-        # DBS3Reader(dbsURL).dbs.serverinfo()
-        from WMCore.Services.Requests import JSONRequests
-        jsonSender = JSONRequests(dbsURL)
-        result = jsonSender.get("/%s" % uri)
-        if not result[1] == 200:
-            raise WMSpecFactoryException("DBS is not connected: %s : %s" % (dbsURL, str(result)))
+        from WMCore.Services.DBS.DBS3Reader import DBS3Reader
```

Comment:
Two comments. First, is there a reason to have the import here instead of in the headers? Second, the function actually makes a DBS call but I guess does not return anything? So the purpose was just to make sure that DBS was online?

What you've written now will pass with the mock emulator no problem which should solve @alexanderrichards is.